### ISSUE TITLE
Is deleting without confirmation too dangerous?

### ISSUE THREAD

**Issue by CharlesNepote** [openfoodfacts/folksonomy_frontend#12]
The delete button goes strait to point and immediately delete the property. Should we need a confirmation or something else?

**Comment by kirtanchandak** [comment]
Yes, I tested this we should give a warning before deleting the property. We can have a modal or alert on the page when user clicks on delete

**Comment by kirtanchandak** [comment]
I can work on this

**Comment by kirtanchandak** [comment]
@teolemon @CharlesNepote let me know your thoughts

**Comment by rxiddhi** [comment]
I also agree @CharlesNepote  @kirtanchandak , is this issue open to work on?

**Comment by alexgarel** [comment]
We will move to web-components so this is not the case to work on it
